packages/turbo/ios/RNVisitableView.swift

```diff
@@ -45,6 +45,10 @@ class RNVisitableView: UIView, RNSessionSubscriber {
     controller.didMove(toParent: reactViewController())
     addSubview(controller.view)
   }
+  
+  override func willMove(toWindow newWindow: UIWindow?) {
+    controller.viewWillAppear(false)
+  }
```

Comment:
Consider adding a comment here on why we have this, as it does look unconventional.

Reply:
👍